telemetry/definitions/commonDefinitions.json

```diff
@@ -3403,17 +3418,17 @@
             ]
         },
         {
-            "name": "codeTransform_logApiError",
+            "name": "codeTransform_logApiResult",
```

Comment:
Combines `logApiError` and `logApiLatency`.

Reply:
Should we leave logApiError/logApiLatency as is for backward compatibility, then deprecate them once no more metrics using these names are being published?

Reply:
That makes sense